pkgs/build-support/fetchurl/builder.sh

```diff
@@ -124,10 +124,10 @@ tryHashedMirrors() {
 # URL list may contain ?. No glob expansion for that, please
 set -o noglob
 
-urls2=
-for url in $urls; do
+urlsResolved=()
```

Comment:
I was initially confused by `` fetchurl: builder.sh: handle `urls` as a Bash array ``'s message. The line wrapping makes it look like the commit is about `` use __structuredAttrs = true and pass curlOptsList directly") ``, however that is a fragment of the referenced commit's summary.

Maybe wrapping it as:

> Clean up leftover for commit cd13136f036deb139311bbbb1d18544af42eec24
> ("fetchurl: use __structuredAttrs = true and pass curlOptsList directly")

would be clearer?